Misc changes for network section (#678)
openstack-k8s-operators/data-plane-adoption: docs_user/assemblies/assembly_configuring-network-for-RHOSO-deployment.adoc

Perform the following tasks to incorporate the existing network configuration:

* Configure {rhocp_long} worker nodes to align VLAN tags and IP Address Management (IPAM) configuration with the existing deployment.
* Configure control plane services to use compatible IP ranges for service and load-balancing IP addresses.
* Configure data plane nodes to use corresponding compatible configuration for VLAN tags and IPAM.

When configuring nodes and services, the general approach is as follows:

> * For IPAM, either reuse subnet ranges from the existing deployment or, specifically due to shortage of free IP addresses in existing subnets, define new ranges for the new control plane services. If you define new ranges, you configure IP routing between the old and new ranges. For more information, see xref:planning-your-ipam-configuration_configuring-network[Planning your IPAM configuration].

Reviewer: ```suggestion
* For IPAM, you can either reuse subnet ranges from the existing deployment or, if there is a shortage of free IP addresses in existing subnets, define new ranges for the new control plane services. If you define new ranges, you configure IP routing between the old and new ranges. For more information, see xref:planning-your-ipam-configuration_configuring-network[Planning your IPAM configuration].
```
